terraform/variables.tf

```diff
@@ -88,7 +88,7 @@ variable "api_key" {
 
 variable "app_count" {
   description = "Number of docker containers to run"
-  default     = 2
+  default     = 1
```

Comment:
I think a new more specific variable like "notifications_api_app_count" that is set in Github actions and deployed in terraform should be used here in the case that we want to change the number of instances in future, to avoid another code change

Reply:
@ssylver93  sounds good! NOTIFICATIONS_API_INSTANCE_COUNT added in terrafrom and github